src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/SafeEvpPKeyHandle.OpenSsl.cs

```diff
@@ -177,5 +193,84 @@ public static SafeEvpPKeyHandle OpenPublicKeyFromEngine(string engineName, strin
 
             return Interop.Crypto.LoadPublicKeyFromEngine(engineName, keyId);
         }
+
+        /// <summary>
+        ///   Open a named public key using a named OpenSSL <code>provider</code>.
```

Comment:
It's weird to put "provider" in codestyle/monospace, given that's not what they're called in OSSL.

Either the plain-language form or the technical form would be fine, but this straddles in a weird place (repeats throughout).

Plain-language
```suggestion
        ///   Open a named public key using a named OpenSSL provider.
```

technical
```suggestion
        ///   Open a named public key using a named <c>OSSL_PROVIDER</c>.
```